src/ports/postgres/modules/deep_learning/test/madlib_keras_automl.sql_in

```diff
@@ -21,6 +21,347 @@
 
 m4_include(`SQLCommon.m4')
 
+\i m4_regexp(MODULE_PATHNAME,
+             `\(.*\)libmadlib\.so',
+              `\1../../modules/deep_learning/test/madlib_keras_iris.setup.sql_in'
+)
+
+m4_changequote(`<!', `!>')
+m4_ifdef(<!__POSTGRESQL__!>, <!!>, <!
+
+--------------------------- MADLIB KERAS AUTOML HYPERBAND TEST CASES ---------------------------
+
+DROP TABLE IF EXISTS iris_train_packed, iris_train_packed_summary;
+SELECT training_preprocessor_dl('iris_train',         -- Source table
+                                       'iris_train_packed',  -- Output table
```

Comment:
Do you mean `iris_data_packed` or `iris_train_packed` for the source (i.e. training) table? I am currently using `iris_train_packed` and I need to have these function calls, otherwise this table is not defined while trying the run hyperband.

Reply:
If you look at the file `madlib_keras_iris.setup.sql_in`, there is already a table `iris_data_packed ` which is equivalent to the `iris_train_packed ` that you are creating here by calling `training_preprocessro_dl()`. Since this file `madlib_keras_iris.setup.sql_in` (L24:27), is executed as part of this test, you don't need to re run  training_preprocessor_dl and validation_preprocessor_dl you can directly use the already preprocessed tables `iris_data_packed` and `iris_data_val`